navigation/index.tsx

```diff
@@ -14,14 +15,19 @@ import { createNativeStackNavigator } from "@react-navigation/native-stack";
 import * as React from "react";
 import { ColorSchemeName, Pressable } from "react-native";
 
-import Colors from 'constants/Colors';
-import useColorScheme from 'hooks/useColorScheme';
-import NewProjectScreen from 'screens/NewProjectScreen';
-import NotFoundScreen from 'screens/NotFoundScreen';
-import ProfileScreen from 'screens/ProfileScreen';
-import SettingsScreen from 'screens/SettingsScreen';
-import { RootStackParamList, RootTabParamList, RootTabScreenProps } from 'types';
-import LinkingConfiguration from 'navigation/LinkingConfiguration';
+import Colors from "constants/Colors";
+import useColorScheme from "hooks/useColorScheme";
+import NewProjectScreen from "screens/NewProjectScreen";
```

Comment:
I wonder why the pre-commit hooks aren't fixing this double quote vs single quote formatting. (is Prettier installed?)

Reply:
Looks like this branch is 38 commits behind master, so it may have not have the pre commit hooks changes.

Reply:
Yeah my bad, I hadn't pulled the  .prettierrc

Reply:
Try `git pull origin master`